app/src/main/res/layout-land/profile_chooser_fragment.xml

```diff
@@ -0,0 +1,119 @@
+          android:minHeight="24dp"
+          android:text="@string/profile_chooser_language"
+          android:textColor="@color/profileChooserGreyTextColor"
+          android:fontFamily="sans-serif"
+          android:textSize="16sp"
+          app:layout_constraintEnd_toStartOf="@+id/profile_chooser_language_icon"
+          app:layout_constraintTop_toTopOf="parent" />
+
+        <ImageView
+          android:id="@+id/profile_chooser_language_icon"
+          android:layout_width="24dp"
+          android:layout_height="24dp"
+          android:layout_marginEnd="36dp"
+          android:fontFamily="sans-serif"
+          android:src="@drawable/ic_language_icon_grey_24dp"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+          android:id="@+id/profile_select_text"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="36dp"
+          android:layout_marginTop="64dp"
+          android:layout_marginEnd="36dp"
+          android:fontFamily="sans-serif"
+          android:text="@string/profile_chooser_select"
+          android:textColor="@color/white"
+          android:textSize="24sp"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toTopOf="parent" />
+
+        <androidx.recyclerview.widget.RecyclerView
+          android:id="@+id/profile_recycler_view"
+          android:layout_width="match_parent"
+          android:layout_height="match_parent"
+          android:layout_marginStart="16dp"
+          android:layout_marginTop="16dp"
+          android:layout_marginEnd="16dp"
+          android:clipToPadding="false"
+          android:fadingEdge="horizontal"
+          android:fadingEdgeLength="72dp"
+          android:orientation="vertical"
+          android:overScrollMode="never"
+          android:paddingBottom="@dimen/bottom_white_space"
+          android:requiresFadingEdge="vertical"
+          android:scrollbars="none"
+          android:tag="profile_recycler_view"
+          app:data="@{viewModel.profiles}"
+          app:layoutManager="androidx.recyclerview.widget.GridLayoutManager"
+          app:layout_constraintBottom_toBottomOf="parent"
+          app:layout_constraintTop_toBottomOf="@id/profile_select_text"
+          app:spanCount="@integer/profile_chooser_span_count" />
+      </androidx.constraintlayout.widget.ConstraintLayout>
+    </androidx.core.widget.NestedScrollView>
+
+    <View
+      android:layout_width="match_parent"
+      android:layout_height="96dp"
+      android:background="@drawable/profile_chooser_gradient"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent" />
+
+    <TextView
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="24dp"
+      android:minHeight="24dp"
```

Comment:
This is clickable right? Min height has to be 48dp to pass accessibility test. Now its passing because there is no click event given to this. Similarly with language.

Reply:
Done.